src/main/java/menion/android/whereyougo/gui/extension/DataInfo.java

```diff
@@ -175,9 +175,7 @@ public DataInfo setImageRight(Bitmap image) {
     }
 
     public Location getLocation() {
-        Location loc = new Location(TAG);
-        loc.setLatitude(value01);
-        loc.setLongitude(value02);
+        Location loc = new Location(value01, value02);
```

Comment:
The 'TAG' string is now unused. Please remove.

Reply:
Done in 2daea5a655714047859cba8bb48e8b816def4518